### ISSUE TITLE
Setting changes in Blink.ini don't work

### ISSUE THREAD

**Issue by hmafinejad** [lurume84/bling-desktop#21]
Hi,

First, thank you for putting together such a great tool! This is such a great complement to the Blink system!

I've made changes in blink.ini file, but the changes don't seem to be taking. For example, I've changed the Output location, but the video downloads are still going to the default location. Same with setting "UseLocalTime=True" doesn't seem to apply. 

Here's what my blink.ini file looks like:

[SyncVideo]
Enabled=true
UseLocalTime=true
Output=S:\SecurityCameras\videos
LastUpdate=2020-01-11T04:59:19+00:00
[SyncThumbnail]
Enabled=true
Interval=7200
Sleep=5
Retries=10
Output=S:\SecurityCameras\Thumbnails
[LiveView]
Output=S:\SecurityCameras\LiveView
UseLocalTime=true

Thanks!

**Comment by lurume84** [comment]
mmm, can you confirm the location of Blink.ini is %userprofile%\Documents\Bling.exe\Blink.ini?

**Comment by hmafinejad** [comment]
Yes, it is in that location. (%userprofile%\Documents\Bling.exe\Blink.ini) 
![Capture](https://user-images.githubusercontent.com/60020588/72770114-c2b34480-3bb1-11ea-9f91-f3cb54347898.JPG)

**Comment by lurume84** [comment]
For the Output setting, you need to put \ at the end. UseLocalTime=true should work, check true is lowercase. This setting should create the mp4 files with %Y-%m-%dT%H:%M:%S+00:00 format in your local time.

I will create a new release in a few days and will review this couple of things. On a separate note, does live view work? Do you have XT or XT2 cameras?

**Comment by hmafinejad** [comment]
I think I know why the Output location wasn't working at all. There was an instance of Blink still running even though I had closed the app. Once I killed that process from Task Manager, and started the app again it started using the new location. 

And yes, I did notice that I do need the \ at the end, otherwise it was appending the year to the end of the parent folder name. That's working well now!

One interesting thing I noticed after setting UseLocalTime=true is that it created the Day folder for the next day and downloaded videos into that folder the night before. For example, yesterday was 1/20, and it created a folder under 2020\January\21 and started to put videos in there from 1/20. So under \21 it created videos with filename "2020-01-20T16_12_48+08_00". (See my screenshot below) Have you seen that?

I have the XT2 cameras and haven't been able to get live view to work. When I try, it goes to a black screen and it just spins. I did check and made sure port 9191 wasn't being used.

Thanks, and looking forward to your new release! I really like this app!

![Capture](https://user-images.githubusercontent.com/60020588/72868796-e81a7e00-3c97-11ea-83c5-16fb570f5f3c.JPG)
